models.go

```diff
@@ -136,6 +134,40 @@ type respUserGet struct {
 	// TODO: extattr external_profile external_position
 }
 
+// respUserGet 读取成员响应
+type respUserGet struct {
+	respCommon
+
+	respUserDetail
+}
+
+// reqUserByDept 部门成员请求
+type reqUsersByDeptID struct {
```

Comment:
nit: the struct name is not consistent with API endpoint name, but I'll fix later.